lib/tasks/url_to_subscriber_list_criteria.rake

```diff
@@ -0,0 +1,7 @@
+desc "Migrate govuk delivery topics to email alert api"
```

Comment:
One final comment: could we make this description sound less like it's doing the actual switchover? Maybe `"Create govuk-delivery topics in email-alert-api as disabled subscriber lists, using data from a CSV file"`?